src/style.css

```diff
@@ -1 +1,66 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  --default-color: #00acdc;
+}
+
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding-right: 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  padding: 10px 0 6px 50px;
+}
+
+.nav_list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_item {
+  display: flex;
```

Comment:
Actually, you don't have a need to use `flex` here.
You may use `display: block`

Reply:
thank you for the correction
I will keep it in mind